draft-gruessing-moq-requirements.md

```diff
@@ -166,6 +173,13 @@ any member of the audience could participate. As this has a much larger total
 number of participants - as many as Live Media Streaming {{lmstream}}, but with
 the bi-directionality of conferencing, this should be considered a "hybrid". There can be additional functionality as well that overlap between the two, such as "live rewind", or recording abilities.
 
+Another consideration is the limits of "human bandwidth" - as the number of
+sources are included into a given session increase, the amount of media that can
+usefully understood by a single person diminishes. To put it more simply - too
+many people talking at once is much more difficult to understand than one person
+speaking at a time. Subsequently this will define some limitations in the number
```

Comment:
I like this, but I wonder if we would get less crap if

> too many people talking at once is much more difficult to understand than one person speaking at a time

was 

N people talking at once is much more difficult to understand than N people speaking one at a time

Reply:
I don't agree - as I said yesterday in the context of concurrent speakers and intelligibility, there's a few states of either nobody speaking, somebody(ies?) speaking, or too many people speaking. The number of which I don't think in this case matters.